### PR TITLE
Fixed typos in comments

### PR DESCRIPTION
**Pull request by PW486**

### CODE REVIEW

File: tensorflow/core/util/saved_tensor_slice.proto [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Protocol buffers for saved tensor slices. It's used for the brain tensor
 // ops checkpoints and the V3 checkpoints in dist_belief.
 
-// A checkpoint file is an sstable. The value for each record is a serialized
+// A checkpoint file is a stable. The value for each record is a serialized
```
Comment (betterenvi) [inline tensorflow/core/util/saved_tensor_slice.proto]: The original should be right.

Comment (betterenvi) [inline tensorflow/core/util/saved_tensor_slice.proto]: The original should be right.

File: tensorflow/core/util/strided_slice_op.cc [hunk 2]
```diff
@@ -218,8 +218,8 @@ Status ValidateStridedSliceOp(
 
   // Step 2: Make a sparse spec into a full index spec
   //
-  // The sparse spec does not corresopnds to the number of dimensions
-  // Make a dense spec that corresponds to thte number of dimensions
+  // The sparse spec does not corresponds to the number of dimensions
```
Comment (betterenvi) [inline tensorflow/core/util/strided_slice_op.cc]: No "s" at the end of "correspond".

File: tensorflow/core/util/saved_tensor_slice.proto (hunk 1 above)
Comment (PW486) [reply tensorflow/core/util/saved_tensor_slice.proto]: thank you!

File: tensorflow/core/util/strided_slice_op.cc [hunk 3]
```diff
@@ -218,8 +218,8 @@ Status ValidateStridedSliceOp(
 
   // Step 2: Make a sparse spec into a full index spec
   //
-  // The sparse spec does not corresopnds to the number of dimensions
-  // Make a dense spec that corresponds to thte number of dimensions
+  // The sparse spec does not correspond to the number of dimensions
+  // Make a dense spec that correspond to the number of dimensions
```
Comment (betterenvi) [inline tensorflow/core/util/strided_slice_op.cc]: "corresponds" here :-)

**Review by jhseu — APPROVED** [review]

**Comment by jhseu** [comment]
Jenkins, test this please

**Pull request merged**